Main/src/New file.kt

```diff
@@ -0,0 +1,5 @@
+class Speed{
```

Comment:
the class is called speed. But it can only print data. Maybe name it like speedPrinter, speedShower etc. and that's decide problem with strange name rate, which do not describe it's meaning